crates/wasmtime-provider/examples/wasmtime-hash-mreplace.rs

```diff
@@ -0,0 +1,101 @@
+#![allow(unused_imports)]
+use console::{style, Term};
+use log::{debug, error, info, warn};
+use serde::{Deserialize, Serialize};
+use smallvec::{smallvec, SmallVec};
```

Comment:
nitpicking: can we use a simple `Vec` from the stdlib, just to keep the example as small as possible?